src/controller.js

```diff
@@ -307,7 +307,7 @@ export default ['$scope', '$element', function ($scope, $element) {
 
   async function createTrellisObjects() {
     // Get viz object
-    if ($scope.currentCube) {
+    if ($scope.currentCube && $scope.layout.prop.vizId) {
```

Comment:
Do we need to check if `prop` is set or even `layout` ? Or that could perhaps not be a scenario in this if statement?

Reply:
I doubt it is possible for them to be null when it gets here. But then again, checking just to be sure has a very low cost.